packages/sdks/web-component/test/descope-wc.test.ts

```diff
@@ -3988,11 +3997,13 @@ describe('web-component', () => {
 
     fireEvent.blur(emailInput);
 
-    await waitFor(() =>
-      expect(
-        (<HTMLInputElement>emailInput).reportValidity,
-      ).toHaveBeenCalledTimes(1),
-    );
+    await waitFor(() => {
+      setTimeout(() => {
```

Comment:
Why do we need setTimeout inside waitFor?

Reply:
Removed timeout and moved the blur trigger inside the `waitFor` block.